src/blocks/theme/semantics/semantics.progress-bar.ts

```diff
@@ -0,0 +1,6 @@
+import { colorBrands } from '../colors/colors.brands';
+
+export const progressBarSemantics = {
```

Comment:
<img width="924" alt="Screenshot 2024-08-01 at 2 43 34 PM" src="https://github.com/user-attachments/assets/fda3dd0c-8443-4459-b54c-68b3faf082f7">

The semantics doesn't matches figma.